### ISSUE TITLE
There is no way to change password for account in app

### ISSUE THREAD

**Issue by al-ign** [nextcloud/ocsms#218]
Recently I was needed to change my password for my NC account - and I didn't found any way to enter a new password in your app, except full app reinstall.
If I missed it somehow - please correct me.
If not - well, it would be nice to have this feature.

Also - there was no indication what there is wrong credentials, just 'Failed to sync' in notifications bar and 'Completed successfully' in popup message.

**Comment by al-ign** [comment]
(regarding topic change - messed up repos, please ignore)
I know about 'Max messages on tab loading' setting, but without async load it is not very useful without async loading.

**Comment by nerzhul** [comment]
i need time to rewrite the app from AngularJS 1 to Vue.JS, i strarted massive code cleanup but it's stalled at this moment, anybody wanting to help can help :)

**Comment by al-ign** [comment]
I'm in no means capable for rewriting from one JS framework to another, but quick googling shows that Angular accepts minus symbol in orderby directive to sort in descending order, so in
https://github.com/nextcloud/ocsms/blob/0c23a10e80d10f5198146e8e4d91812998ed14dd/templates/main.php#L83
'date' could be changed to '-date' and this will change list sort to descending order.
Documentation: https://docs.angularjs.org/api/ng/filter/orderBy

**Comment by nerzhul** [comment]
@al-ign yeah, but it means you load all messages too, atm we load recent messages, and you can change ordering on the loaded message, we should also change loading order, not just triage order :)

**Comment by al-ign** [comment]
> we should also change loading order
To support async/on-demand load - probably yes, you should know better.

As for now - it works, I changed to '-orderBy' in my installation - and I'm okay with the result for now.

**Comment by nerzhul** [comment]
@al-ign it's a workaround but it works 📦

**Comment by al-ign** [comment]
Well, I got what I need :godmode:

Close, or leave as remainder to add reverse sort? Sometime after rewrite to Vue. In a year or two...

**Comment by toolstack** [comment]
I noticed there actually some JavaScript in the code to automatically scroll to the bottom of the windows when a conversation is loaded, but it's broken due to an incorrect div id.  I'll create a PR to resolve it.

**Comment by nerzhul** [comment]
it's merged, thanks @toolstack
